### ISSUE TITLE
hyperkube should support running as kubectl

### ISSUE THREAD

**Issue by thockin** [kubernetes/kubernetes#16436]
This would make demos easier.

**Comment by resouer** [comment]
I'm also expecting this. 

What's the preferred behavior in this case? Just run as a container? Or use a hyperkube container to install kubectl to $PATH?

**Comment by thockin** [comment]
Yeah, the fun part is that kubectl expects to find your credentials and
things.  Needs some design to make this usable.

**Comment by resouer** [comment]
**Option 1**

`docker run -v <path_need> hyperkube kubectl get pods`

This seems verbose ...

**Option 2**
```
docker run --rm -v /usr/local/bin:/target hyperkube
alias kubectl='hyperkube kubectl'
```
Em, looks better?

**Comment by brendandburns** [comment]
```console
alias kubectl='docker exec <apiserver-container> /hyperkube kubectl'
```

**Comment by brendandburns** [comment]
This was closed by #17207

**Comment by denis-kalinin** [comment]
```bash
alias kubectl=`docker ps | awk '$2=="quay.io/coreos/hyperkube"{print $1; exit}' | awk 'END {printf "docker exec %s /hyperkube kubectl", $0}'`
```
